src/onnx/parse_lstm.cpp

```diff
@@ -194,7 +122,10 @@ struct parse_lstm : op_parser<parse_lstm>
             MIGRAPHX_THROW("LSTM: incorrect direction attribute");
         }
 
+        // set default activation functions
         std::vector<std::string> vec_names = {"sigmoid", "tanh", "tanh"};
+        lstm_actv_functions(dirct, vec_names);
```

Comment:
You could easily get rid of lstm_actv_functions, by default filling line 126, above. Just as in parse_rnn.cpp.

Reply:
@lakhinderwalia  It is not completely the same as in rnn default is 'tanh','tanh' (as for bidirectional) and here it is 'sigmoid','tanh','tanh' as for one-directional. However, I did want to get rid of the lstm_actv_functions and place the code in the parse function, but the tidy check was complaining about exceeding recommended complexity (having 16 branches instead of maximal 15), so I left that part in this shortened version of  lstm_actv_functions. If you prefer changing it somehow, it is not an issues, but I didn't find a way not to exceed it.

Reply:
This is not a big issue. Simplification of the code can be done in many ways, but that shouldn't block this PR at this point, IMO.